iothub_client/tests/iothubclient_uploadtoblob_e2e/iothubclient_uploadtoblob_e2e.c

```diff
@@ -240,13 +240,13 @@ IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_RESULT uploadToBlobGetDataEx(IOTHUB_CLIENT_FI
 
     UPLOADTOBLOB_CALLBACK_STATUS* callbackStatus = (UPLOADTOBLOB_CALLBACK_STATUS*)context;
     ASSERT_ARE_EQUAL(UPLOADTOBLOB_CALLBACK_STATUS, *callbackStatus, UPLOADTOBLOB_CALLBACK_PENDING);
-    ASSERT_ARE_EQUAL(int, (int)result, (int)FILE_UPLOAD_OK);
```

Comment:
Anything we can do in e2e to verify we're correctly sending the correlationID? Is there any information flown back from hub after the Blob upload has failed?

Reply:
We do since the service rejects it if the correlation id is wrong.